projects/kit/components/input-range/input-range.component.ts

```diff
@@ -92,29 +84,19 @@ export class TuiInputRange extends TuiControl<readonly [number, number]> {
         this.contentStart() === this.content()[0] ? this.content()[1] : '',
     );
 
-    @Input()
-    public min = 0;
-
-    @Input()
-    public max = 100;
-
-    @Input()
-    public step = 1;
-
-    @Input()
-    public segments = 1;
-
-    @Input()
-    public keySteps: TuiKeySteps | null = null;
-
-    @Input({transform: (x: readonly [string, string] | null) => x ?? ['', '']})
-    public prefix: readonly [string, string] = [this.options.prefix, this.options.prefix];
+    public readonly min = input(0);
+    public readonly max = input(100);
+    public readonly step = input(1);
+    public readonly segments = input(1);
+    public readonly keySteps = input<TuiKeySteps>();
```

Comment:
Why not ` TuiKeySteps | null` (as it was before) ?

Reply:
So that it becomes just an optional input without explicit `null` as a value